packages/web/src/common/store/pages/audio-rewards/sagas.ts

```diff
@@ -308,8 +320,24 @@ function* claimSingleChallengeRewardAsync(
     )
     yield* put(setUserChallengesDisbursed({ challengeId, specifiers: claimed }))
 
+    // Ignore other errors and return early if there are AAO errors.
+    const aaoErrors = results.filter(
+      (r): r is AAOErrorResult => 'aaoErrorCode' in r
+    )
+    if (aaoErrors.length > 0) {
+      for (const error of aaoErrors) {
+        console.error(
+          `AAO attestation failed for specifier: ${error.specifier} for amount: ${error.amount}`
+        )
+      }
+      // Only consider the first AAO error code; others are likely to be the same anyway for the same user.
+      const aaoErrorCode = aaoErrors.map((r) => r.aaoErrorCode)[0]
+      return { aaoErrorCode }
```

Comment:
why not throw `new Error(aaoErrorCode)` here?

Reply:
1. would have then to play with typeof error === 'number'
2. our current web TS setup does not allowing throwing non Error items (included this in my description)

Reply:
nvm. my point 2. is only relevant in sdk code, not web